internal/http/processor.go

```diff
@@ -152,63 +152,65 @@ func (p processor) ProcessSeasonPack(w netHTTP.ResponseWriter, r *netHTTP.Reques
 	clientIndex := findClientIndex(p.cfg.Config, p.req.ClientName)
 
 	if clientIndex == -1 {
-		p.log.Info().Msgf("client name %q not found in config file, using first client defined in config: %q ", p.req.ClientName, p.cfg.Config.Clients[0].Name)
 		// default to first client in config
+		p.log.Info().Msgf("client not found in config: %q", p.req.ClientName)
+		p.log.Info().Msgf("using first client defined in config: %q ", p.cfg.Config.Clients[0].Name)
+
 		clientIndex = 0
 	}
 
 	if len(p.req.Name) == 0 {
-		p.log.Error().Msgf("no title passed")
-		netHTTP.Error(w, fmt.Sprintf("no title passed"), 469)
+		p.log.Error().Msgf("error getting announce name")
+		netHTTP.Error(w, fmt.Sprintf("error getting announce name"), 469)
 		return
 	}
 
 	if err := p.getClient(clientIndex); err != nil {
-		p.log.Error().Err(err).Msgf("unable to get client")
-		netHTTP.Error(w, fmt.Sprintf("unable to get client: %q", err), 471)
+		p.log.Error().Err(err).Msgf("error getting client")
+		netHTTP.Error(w, fmt.Sprintf("error getting client: %q", err), 471)
 		return
 	}
 
 	mp := p.getAllTorrents(clientIndex)
 	if mp.err != nil {
-		p.log.Error().Err(mp.err).Msgf("unable to get torrents")
-		netHTTP.Error(w, fmt.Sprintf("unable to get torrents: %q", mp.err), 468)
+		p.log.Error().Err(mp.err).Msgf("error getting torrents")
+		netHTTP.Error(w, fmt.Sprintf("error getting torrents: %q", mp.err), 468)
 		return
 	}
 
 	requestrls := entry{r: rls.ParseString(p.req.Name)}
 	v, ok := mp.e[utils.GetFormattedTitle(requestrls.r)]
 	if !ok {
-		p.log.Info().Msgf("unique submission: %q", p.req.Name)
-		netHTTP.Error(w, fmt.Sprintf("unique submission: %q", p.req.Name), 200)
+		p.log.Info().Msgf("no matching releases in client %q: %q", p.cfg.Config.Clients[clientIndex].Name, p.req.Name)
+		netHTTP.Error(w, fmt.Sprintf("no matching releases in client %q: %q", p.cfg.Config.Clients[clientIndex].Name, p.req.Name), 200)
```

Comment:
`%q` is used for errors, `%s` for strings. There's a couple of others like `%d` for numbers and if you want the value then `%v`.

These above should be changed to `%s`.